src/Datagrid/Datagrid.php

```diff
@@ -241,6 +241,11 @@ public function setValue($name, $operator, $value)
         ];
     }
 
+    public function setSimpleValue($name, $value)
```

Comment:
`DatagridInterface` should add this method too, and this should be commented with its corresponding "NEXT_MAJOR" message.
```suggestion
    public function setSimpleValue(string $name, $value): void
```